WoWsShipBuilder.Web/Data/ShipBuildContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using WoWsShipBuilder.Core.BuildCreator;
+using WoWsShipBuilder.Core.DataContainers;
+using WoWsShipBuilder.DataStructures;
+
+namespace WoWsShipBuilder.Web.Data;
+
+public sealed record ShipBuildContainer(Ship Ship, Build? Build, Guid Id, IEnumerable<int>? ActivatedConsumableSlots, bool SpecialAbilityActive, ShipDataContainer? ShipDataContainer)
```

Comment:
This class has no web exclusive features. It should be in one of the shared project (either ViewModels or Core), so it can be used by both desktop and web. No reason to have a duplicate when it's not needed.